lib/vagrant-berkshelf/action/share.rb

```diff
@@ -22,7 +22,7 @@ def call(env)
             value = chef.config.send(:prepare_folders_config, env[:berkshelf].shelf)
 
             @logger.debug "Setting cookbooks_path = #{value.inspect}"
-            chef.config.cookbooks_path = value
+            chef.config.cookbooks_path += value
```

Comment:
If we are going to do this, I think we need to prepend the array with the value, not append it as you have here. The cookbooks in the Berkshelf should take precedence over the cookbooks in other directories, so you should do something like:

```ruby
chef.config.cookbooks_path = [value] + Array(chef.config.cookbooks_path)
```